Add installation instructions for Composer

Saboteur777 opened the pull request.

Saboteur777 committed:
Add installation instructions for Composer

speder commented:
1. Append "ether/seo": "^3.0.1" to the "require" hash of composer.json
2. composer update
3. Install via CP in/admin/settings/plugins

Saboteur777 committed:
Alternative installation instructions added

Updated installation instructions with an alternative by https://github.com/speder (https://github.com/ethercreative/seo/pull/61#issuecomment-369333139)

Saboteur777 reviewed (commented):
@speder Your proposal has been added.
